### PR TITLE
CASMTRIAGE-6053 Update CSM_ARTI_DIR and ceph mount

### PR DESCRIPTION
**Pull request by ganeshs-hpe**
# Description

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 1]
```diff
@@ -83,6 +83,29 @@ export CSM_RELEASE=1.4.0
 export CSM_ARTI_DIR="/etc/cray/upgrade/csm/csm-${CSM_RELEASE}/tarball/csm-${CSM_RELEASE}"
 ```
 
+> **`NOTES:`**
+>
+> - If directory `/etc/cray/upgrade/csm/` is empty, create empty directory at same path. Download and extract CSM tarball to that directory.
```
Comment (bo-quan) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md]:
```suggestion
> - If the `/etc/cray/upgrade/csm/` directory is empty, create an empty directory at the same path. Download and extract CSM tarball to that directory.
```

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 2]
```diff
@@ -83,6 +83,29 @@ export CSM_RELEASE=1.4.0
...
+> - Update value of `CSM_ARTI_DIR` with newly created directory above.
```
Comment (bo-quan) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md]:
```suggestion
> - Update the value of `CSM_ARTI_DIR` with the newly created directory above.
```

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 3]
```diff
@@ -83,6 +83,29 @@ export CSM_RELEASE=1.4.0
...
+> - Ensure directory `/etc/cray/upgrade/csm/` is ceph mount using command as below:
```
Comment (bo-quan) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md]:
```suggestion
> - Ensure the `/etc/cray/upgrade/csm/` directory is `ceph` mount using the command below (its output should show `ceph` as the type):
```

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 4]
```diff
@@ -83,6 +83,29 @@ export CSM_RELEASE=1.4.0
...
+
+```bash
+mount | grep /etc/cray/upgrade/csm
+admin-tools@<hostname>=/ on /etc/cray/upgrade/csm type ceph (rw,relatime,name=admin-tools,secret=<hidden>,ms_mode=prefer-crc,acl,mon_addr=<ip:port>)
```
Comment (bo-quan) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md]: I would remove the output line as it is a little confusing to include the output in the bash command block.

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 5]
```diff
@@ -83,6 +83,29 @@ export CSM_RELEASE=1.4.0
...
+```
+
+> - If directory `/etc/cray/upgrade/csm/` is not ceph mount, use directory which is ceph mount as specified in file `/etc/fstab`. Sample example below:
```
Comment (bo-quan) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md]: I'm not sure about the right procedure - probably needs to consult with Craig.

**Review by bo-quan — CHANGES_REQUESTED** [review]

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md (hunk 4 above)
Comment (ganeshs-hpe) [reply operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md]: Line removed. Thanks.

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md (hunk 5 above)
Comment (ganeshs-hpe) [reply operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md]: Line removed. Thanks.

**Comment by ganeshs-hpe** [comment]
@bo-quan Thanks for the review. I have updated accordingly. Kindly review to ensure it meets the specs.

**Review by bo-quan — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**